Project-Hotpot/Project-Hotpot/Algorithm/randomSong.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  randomSong.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/11/22.
+//
+
```

Comment:
Consider using Classes unless using only functional programming is a good swift pattern (cc @toby1236 )
Additionally think about names of Files / Classes. Generally 'RandomSong' seems like name of a model class that holds data or class that represents 'a song' but looks like this could be your Algorithm core. A better name suggestion would be "SongAlgorithm"